config/locales/defaults/en.yml

```diff
@@ -65,7 +65,7 @@ en:
       context: '%{percent_users_searched}% of users searched from the page'
       unit: 'search terms'
       data_source: google_analytics
-      external_link: 'See search terms in Google Analytics (opens in new tab)'
```

Comment:
We could technically remove this definition entirely now, as it's not used anywhere. It was used in
https://github.com/alphagov/content-data-admin/blob/main/app/views/metrics/_metric_section.html.erb#L17 but since external_link in `nil` it won't get rendered. However there's a pattern of leaving things blank in this file so let keep it as it is.